common/OpTestHMC.py

```diff
@@ -914,6 +911,32 @@ def run_command(self, i_cmd, timeout=60):
         :param timeout: number, time out in seconds
         '''
         return self.ssh.run_command(i_cmd, timeout)
+   
+    def is_perfcollection_enabled(self):
+        '''
+        Get pef collection allowed in hmc profile
+
+        :returns: Ture if  allow_perf_collection in hmc otherwise false
+        '''
+
+        rc = self.run_command("lssyscfg -m %s -r lpar --filter lpar_names=%s -F allow_perf_collection"
+                                % (self.mg_system, self.lpar_name))
+        if  rc:
+            return True
+	return False
+
+    def hmc_perfcollect_configure(self, enable=True):
+        '''
+        Enable/Disable perfcollection from HMC
+        '''
+        # Set perf collection profile  value using HMC command
+        cmd = ('chsyscfg -r lpar -m %s -i "name=%s, allow_perf_collection=' %
```

Comment:
there is only one double qoute .. is this a typo ?

Reply:
yes that is correct 

```
2024-05-08 14:52:29,484:op-test.common.OpTestHMC:wait_lpar_state:INFO:Current state: Shutting Down
[console-expect]#lssyscfg -m system -r lpar --filter lpar_names=system-lp13-CR -F state
lssyscfg -m system -r lpar --filter lpar_names=system-lp13-CR -F state
Not Activated
[console-expect]#echo $?
echo $?
0
[console-expect]#lsrefcode -m system -r lpar --filter lpar_names=system-lp13-CR -F refcode
lsrefcode -m system -r lpar --filter lpar_names=system-lp13-CR -F refcode
00000000
[console-expect]#echo $?
echo $?
0
2024-05-08 14:52:44,765:op-test.common.OpTestHMC:wait_lpar_state:INFO:Current state: Not Activated
[console-expect]#lssyscfg -m system -r lpar --filter lpar_names=system-lp13-CR -F allow_perf_collection
lssyscfg -m system -r lpar --filter lpar_names=system-lp13-CR -F allow_perf_collection
1
[console-expect]#echo $?
echo $?
0
2024-05-08 14:52:59,910:op-test.testcases.MachineConfig:LparSetup:INFO:System is already booted with perf collection profile enabled
[console-expect]#lssyscfg -m system -r lpar --filter lpar_names=system-lp13-CR -F state
lssyscfg -m system -r lpar --filter lpar_names=system-lp13-CR -F state
Not Activated
[console-expect]#echo $?
echo $?
0
[console-expect]#lsrefcode -m system -r lpar --filter lpar_names=system-lp13-CR -F refcode
lsrefcode -m system -r lpar --filter lpar_names=system-lp13-CR -F refcode
00000000
[console-expect]#echo $?
echo $?
0
[console-expect]#lssyscfg -m system -r lpar --filter lpar_names=system-lp13-CR -F state
lssyscfg -m system -r lpar --filter lpar_names=system-lp13-CR -F state
Not Activated
[console-expect]#echo $?
```